app/validation/required.py

```diff
@@ -0,0 +1,22 @@
+from app.validation.i_validator import IValidator
+from app.validation.validation_result import ValidationResult
+
+
+class Required(IValidator):
+
```

Comment:
For consistency, this probably should be named the same as whatever you change the Integer validator to.